draft-ietf-quic-transport.md

```diff
@@ -889,23 +908,24 @@ Packets protected with 0-RTT keys use a type value of 0x7C.  The connection ID
 fields for a 0-RTT packet MUST match the values used in the Initial packet
 ({{packet-initial}}).
 
-The client can send 0-RTT packets after receiving an Initial
-{{packet-initial}} or Handshake ({{packet-handshake}}) packet, if that
+The client can send further 0-RTT packets after receiving an Initial
+({{packet-initial}}) or Handshake ({{packet-handshake}}) packet, if that
 packet does not complete the handshake.  Even if the client receives a
 different connection ID in the Handshake packet, it MUST continue to
 use the same Destination Connection ID for 0-RTT packets, see
 {{connection-id-encoding}}.
 
-The version field for protected packets is the current QUIC version.
+The version field for protected packets contains the current QUIC version.
```

Comment:
I'm not sure this is correct?

Reply:
I think it is. `The version field … contains the`